### ISSUE TITLE
5.0.0: test suite warnings

### ISSUE THREAD

**Issue by kloczek** [borntyping/python-colorlog#102]
I'm not sure is it OK to display warnigns about version which is not used so better is to flag it :P
```console
+ PYTHONPATH=/home/tkloczko/rpmbuild/BUILDROOT/python-colorlog-5.0.0-2.fc35.x86_64/usr/lib64/python3.8/site-packages:/home/tkloczko/rpmbuild/BUILDROOT/python-colorlog-5.0.0-2.fc35.x86_64/usr/lib/python3.8/site-packages
+ /usr/bin/python3 -Bm pytest -ra
=========================================================================== test session starts ============================================================================
platform linux -- Python 3.8.9, pytest-6.2.3, py-1.10.0, pluggy-0.13.1
rootdir: /home/tkloczko/rpmbuild/BUILD/python-colorlog-5.0.0, configfile: setup.cfg
plugins: forked-1.3.0, shutil-1.7.0, virtualenv-1.7.0, asyncio-0.14.0, expect-1.1.0, cov-2.11.1, mock-3.5.1, httpbin-1.0.0, xdist-2.2.1, flake8-1.0.7, timeout-1.4.2, betamax-0.8.1, pyfakefs-4.4.0, freezegun-0.4.2, flaky-3.7.0, hypothesis-6.9.1
collected 27 items

colorlog/tests/test_colorlog.py ...........                                                                                                                          [ 40%]
colorlog/tests/test_config.py ..                                                                                                                                     [ 48%]
colorlog/tests/test_escape_codes.py .........                                                                                                                        [ 81%]
colorlog/tests/test_example.py .                                                                                                                                     [ 85%]
colorlog/tests/test_exports.py .                                                                                                                                     [ 88%]
colorlog/tests/test_logging.py ...                                                                                                                                   [100%]

============================================================================= warnings summary =============================================================================
colorlog/__init__.py:51
  /home/tkloczko/rpmbuild/BUILD/python-colorlog-5.0.0/colorlog/__init__.py:51: UserWarning: Colorlog 6.0.0 will require Python 3.5 or above. Pin 'colorlog<5' to your dependencies if you require compatibility with older versions of Python. See https://github.com/borntyping/python-colorlog#status for more information.
    warnings.warn(

-- Docs: https://docs.pytest.org/en/stable/warnings.html
====================================================================== 27 passed, 1 warning in 0.27s =======================================================================
```

**Comment by borntyping** [comment]
This is already fixed in [5.0.1](https://pypi.org/project/colorlog/5.0.1/), see https://github.com/borntyping/python-colorlog/commit/884be6c6eadc0473b6d844be78853f3ce316068c.

**Comment by kloczek** [comment]
Thank you :)
